charts/app-config/values-integration.yaml

```
@@ -1260,6 +1260,11 @@ govukApplications:
             secretKeyRef:
               name: govuk-chat-postgres
               key: DATABASE_URL
+        - name: NOTIFY_API_KEY
```

```suggestion
        - name: GOVUK_NOTIFY_API_KEY
```

Sorry nit pick comment: looking at the rest of this file there seems to be a convention to prefix with GOVUK_ before NOTIFY env vars.